docs/management/connectors/action-types/servicenow.asciidoc

```diff
@@ -7,6 +7,64 @@
 
 The ServiceNow ITSM connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow incidents.
 
+[float]
+[[servicenow-itsm-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow ITSM connector or <<servicenow-itsm-connector-update, updating an existing one>>:
+
+* Install Elastic for ITSM from the https://store.servicenow.com/sn_appstore_store.do#!/store/home[ServiceNow Store].
```

Comment:
With a very high probability this will be the link for ITSM in the store: https://store.servicenow.com/sn_appstore_store.do#!/store/application/7148dbc91bf1f450ced060a7234bcb88

Reply:
Thanks for providing this! I'll replace the generic link with this one.